plumbum/machines/local.py

```diff
@@ -233,6 +233,8 @@ def __getitem__(self, cmd):
             return LocalCommand(cmd)
 
         if not isinstance(cmd, RemotePath):
+            # handle "path-like" (pathlib.Path) objects
+            cmd = str(cmd)
```

Comment:
```suggestion
            cmd = os.fspath(cmd)
```

Use `os.fspath`. This would support custom objects that have differing `__str__` and `__fspath__`. And it would not try to apply on things that are not path-like.

Reply:
nice one :)
done